charts/mattermost-operator/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: mattermost-operator
+description: A Helm chart for Kubernetes
```

Comment:
vvvv minor: Should change a bit the description

Reply:
Thanks @stafot, fixed :)